.pylintrc

```diff
@@ -0,0 +1,7 @@
+[MAIN]
+
+ignore-paths=^conbench/app/.*$,
+             ^conbench/api/.*$,
+             ^conbench/entities/.*$,
+             ^conbench/tests/.*$,
```

Comment:
I picked this technique: https://github.com/pylint-dev/pylint/issues/2686#issuecomment-864585778